cmd/otelcontribcol/go.mod

```
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.21.0
 
-toolchain go1.21.11
+toolchain go1.22.4
```

```suggestion
toolchain go1.21.11
```
This is causing the GitHub action `make genotelcontribcol` failure. Your local environment is running the command on `go1.22.4`, while GitHub is running against `go1.21.11`. When the GitHub action generates this file it generates it with the older toolchain version. When you run it locally, it generates it with your local golang version.